src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
```

Comment:
When assigning `key` to `invertedRobot[robot[key]]`, ensure that you are storing the original key as a string. If the key is a number, it will be coerced into a string, which might not be intended. If you want to preserve the type, you might need to handle it differently.